addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -59,6 +59,12 @@ Updated by Wliu, Chris, Lawd, and Carge after Powerlord quit FF2
 #define MONOCULUS "eyeball_boss"
 #define DISABLED_PERKS "toxic,noclip,uber,ammo,instant,jump,tinyplayer"
 
+#define OVER_9000 "saxton_hale/9000.wav"
+#define OVER_9000_PREFIX "sound/saxton_hale/9000.wav"
+
+// In rare cases, some server operators may need to undef this.
+#define FILECHECKS_ON
```

Comment:
Transit of Venus has a rare case where his SM install can't find the files despite them existing.